### DISCUSSION TITLE
How to fill database with data when running locally

### DISCUSSION THREAD

**Discussion by F-Goncalves** [microsoft/dotnet-podcasts#158] [Q&A] [answered]
Hi,

I have deployed the apps to my local docker instance and then ran the MAUI app. However, the app does not list any podcasts, since the locally running api returns 0 podcasts. Now I am wondering how I get any initial data into the local database.

Is there anything provided for that? 
I don't think I can access the public api or am I missing something?

**Comment by F-Goncalves** [comment, marked as answer]
I don't know why I didn't have data in it. But when I tried again and made sure the updaterworker is running it worked.

**Reply by jamesmontemagno** [reply]
Ah yes, it is possible that the updaterworker failed to start when first booting it up based on when things are created, but just boot it up again and the seed data will inflate.
